docker-compose-test.yml

```diff
@@ -14,7 +14,7 @@ services:
     - hmpps_int
     container_name: wiremock
     restart: always
-    command: "--global-response-templating --verbose"
+    command: "--global-response-templating --enable-stub-cors" # add command `--verbose` to output verbose request matching
```

Comment:
Because of the way some of the css and fonts are served via wiremock (in relation to how the page itself is served) there is a CORS issue, so needed to enable CORS headers on wiremock stubs
Also removed `--verbose` as it was getting very very noisy!